packages/picasso.js/src/core/chart-components/property-resolver/index.js

```diff
@@ -18,26 +18,44 @@ function isPrimitive(x) {
 }
 
 /**
+ * Callback function
  * @callback datumAccessor
- * @param {DatumExtract} d
+ * @param {DatumExtract} d datum
+ * @example
+ * (d) => Math.min(0, d.value);
  */
 
 /**
  * @typedef {object} DatumConfig
- * @property {string} [scale]
- * @property {datumAccessor} fn
- * @property {string} ref - A reference to a DatumExtract property
+ * @property {string} [scale] - Name of a scale
+ * @property {datumAccessor} [fn]
+ * @property {string} [ref] - A reference to a DatumExtract property
+ * @example
+ * // Implicitly resolve the datum by passing a referenced data through the scale
+ * {
+ *  scale: '<name-of-scale>',
+ *  ref: '<data-property>'
+ * }
+ *
+ * // or explictly resolve the datum using callback function
+ * {
+ *  scale: '<name-of-scale>',
+ *  fn: (d) => Math.min(0, d.value);
+ * }
```

Comment:
consider removing `scale: '<name-of-scale>',`
I don't think is relevant for this example.

and I think `fn: (d) => Math.min(0, d.datum.x.value);` is a more realistic function

Reply:
Fixed